manifests/server/config_entry.pp

```diff
@@ -5,9 +5,9 @@
 # @param path Path for postgresql.conf
 #
 define postgresql::server::config_entry (
-  Enum['present', 'absent']             $ensure = 'present',
-  Optional[Variant[String[1], Integer]] $value  = undef,
-  Variant[Boolean, String[1]]           $path   = false
+  Enum['present', 'absent']                    $ensure = 'present',
+  Optional[Variant[String[1], Integer, Float]] $value  = undef,
```

Comment:
Elsewhere it was suggested that there's `Numeric` for both Integer and Float. https://www.puppet.com/docs/puppet/7/lang_data_abstract.html#lang_data_abstract_flexible-variant-data-type states `Variant[Integer, Float]` is equivalent to `Numeric`.

Reply:
Thank you @ekohl, I like 👍